ImagesService/database/helpers/generatePhotos.js

```diff
@@ -0,0 +1,54 @@
+const Promise = require('bluebird');
+const { getUnsplashRooms, getUnsplashHosts, getUnsplashReviewers } = require('./getUnsplash.js');
+const faker = require('faker');
+
+async function allUrls() {
+  let allPhotos = {};
+  let roomUrls = [];
+  let rooms = await getUnsplashRooms('living,indoors,room')
```

Comment:
Now, for all 100 id same amount of pics & same pics [mostly] will be generated; So it will not reflect in the UI for random id when we pass as url params

To overcome this - we can randomly generate different pics, like below

Here you can try randomly pass either anyone value by storing in an array
```
let rooms = ['living', 'indoors', 'room', 'interior', 'furniture'];
var getRandomRoomPic = () => {
  var index = Math.floor(Math.random() * placeType.length);
  return rooms[index];
}
```

let randomRoom = getRandomRoomPic();
getUnsplashRooms(randomRoomPic);


Also, just like how u have handled hostPic by generating only one, similarly we can do for the other two.
If u add another argument let's say 'max' in getUnsplash.js file
can Get random number of pics for each id like so,
let max = faker.random.number({min: 1, max: 12}); // this generates random number between 1 to 12
getUnsplashRooms(randomRoomPic, max);

inside the getUnsplash.js file can pass this max argument to the target url like,
```
getUnsplashRooms = (query = 'living,indoors,room', max);
https://api.unsplash.com/search/photos/?query=${query}&page=1&per_page=${max}&orientation=landscape&client_id=${process.env.UNSPLASH_API_KEY}


Apply this for getUnsplashReviewers.

Reply:
Ah yes, the situation with the same number and results for images is something I was planning to resolve, but I can see that I didn't fully understand your ideas with using max before. Now I understand. Also, thanks for showing me faker.random.number. That's super helpful. I will implement this.